pkg/k8s/node.go

```diff
@@ -216,6 +217,16 @@ func GetNode(c kubernetes.Interface, nodeName string) (*corev1.Node, error) {
 	return c.CoreV1().Nodes().Get(context.TODO(), nodeName, metav1.GetOptions{})
 }
 
+// IsProviderKind returns true if the given node with nodeName is running in
+// a Kind cluster.
+func IsProviderKind(c kubernetes.Interface, nodeName string) (bool, error) {
+	node, err := GetNode(c, nodeName)
+	if err != nil {
+		return false, err
+	}
```

Comment:
replace this with something else, a `GetNode` was already performed before we reach this line.

Reply:
Where do you suggest to catch the result of a previous invocation of `GetNode`?

Reply:
on line 511 of `daemon/cmd/daemon.go`

Reply:
Thanks, done.